src/content/plugins/define-plugin.md

```diff
@@ -104,13 +104,22 @@ It is possible to define variables with values that rely on files and will be re
 Arguments:
 
 - The first argument of the `webpack.DefinePlugin.runtimeValue` is a `function` that should return the value to be assigned to the definition.
```

Comment:
Let's add the signature https://github.com/webpack/webpack/blob/master/types.d.ts#L2408 for the first argument as well.